gui/src/streamsession.cpp

```diff
@@ -257,8 +295,12 @@ void StreamSession::InitAudio(unsigned int channels, unsigned int rate)
 	audio_format.setSampleSize(16);
 	audio_format.setCodec("audio/pcm");
 	audio_format.setSampleType(QAudioFormat::SignedInt);
+	
+	Settings user_settings;
+	QAudioDeviceInfo audio_device_info = user_settings.GetAudioOutDevice();
```

Comment:
No need for local user_settings:
```suggestion
	QAudioDeviceInfo audio_device_info = settings->GetAudioOutDevice();
```

Reply:
I actually tried that earlier but was surprised to get:

```
/home/pi/chiaki/chiaki/gui/src/streamsession.cpp:300:39: error: ‘settings’ was not declared in this scope
  QAudioDeviceInfo audio_device_info = settings->GetAudioOutDevice();

```
StreamSessionConnectInfo  does have a 'session' member.
But I can't see a StreamSessionConnectInfo type member of StreamSession, even if it gets initialized with one. It seems to be just used locally in init?